### ISSUE TITLE
Grad College Thesis Calendar not accessible

### ISSUE THREAD

**Issue by e-marie-w** [uiowa/uiowa#8497]
Grad college reached out about https://grad.uiowa.edu/academics/thesis-defenses, it is being flagged as not accessible in SiteImprove.
